jetty-ee10/jetty-ee10-quickstart/src/main/java/org/eclipse/jetty/ee10/quickstart/QuickStartDescriptorProcessor.java

```diff
@@ -70,6 +74,8 @@ public void start(WebAppContext context, Descriptor descriptor)
     public void end(WebAppContext context, Descriptor descriptor)
     {
         _originAttributeName = null;
+        if (_mountedResources != null)
```

Comment:
This can't be the point at which we close the mounts. This method is called when we've finished parsing the `quickstart-web.xml`, _but_ we have passed a `Collection<Resource>` based on these mounts out of this class via the context attribute `METAINF_RESOURCE`.